plugins/ingest-attachment/src/main/java/org/opensearch/ingest/attachment/AttachmentProcessor.java

```diff
@@ -134,9 +135,10 @@ public IngestDocument execute(IngestDocument ingestDocument) {
         }
 
         if (properties.contains(Property.LANGUAGE) && Strings.hasLength(parsedContent)) {
-            // TODO: stop using LanguageIdentifier...
-            LanguageIdentifier identifier = new LanguageIdentifier(parsedContent);
-            String language = identifier.getLanguage();
+            OptimaizeLangDetector langDetector = new OptimaizeLangDetector();
```

Comment:
It looks like initialization could be an expensive operation, should `OptimaizeLangDetector` + `loadModels()` be done only once during `AttachmentProcessor` initialization?